tools/api-markdown-documenter/src/index.ts

```diff
@@ -31,8 +31,8 @@ export * from "./documentation-domain";
 
 export {
 	createDocumentWriter,
-	getContextWithDefaults as getMarkdownRenderContextWithDefaults,
 	DocumentWriter,
+	getContextWithDefaults as getMarkdownRenderContextWithDefaults,
```

Comment:
Is there some reason you're exporting with a different name rather than renaming the internal name?

Reply:
I'm planning on adding a separate HTML renderer in the future, and I figured shorter names in the internal scope would be preferable. But I could go either way if we have recommended practices around this sort of thing.

Reply:
Probably just a pet peeve. I find it hard doing the translation when I write both code that uses the library and code within the library. :) Carry on.